autogen/paths.py

```diff
@@ -15,12 +15,13 @@
 from __future__ import absolute_import, unicode_literals
 
 import os
+import shutils
+
```

Comment:
I mean in the import here

Reply:
Thanks. It should be `shutil`. 
https://docs.python.org/3/library/shutil.html